sensor_msgs/Image image
float32[] coordinates_2d
float32[] coordinates_3d